test/registrationTest.js

```diff
@@ -6,39 +6,50 @@ chai.use(require('chai-http'));
 
 const app = require('../index.js'); // Our app
 
-var body = {
-    first:"test",
-    last:"test1",
-    username: "fakeuser",
-    email:"test@fake.email.com",
-    password:"test"
+var registrationBody = {
+    "first":"test",
+    "last":"test",
+    "email":"test@fake.email.com",
+    "password":"test"
 };
 
-describe('Registration endpoint', function () {
-
-    this.timeout(5000); // How long to wait for a response (ms)
-
-    before(function () {
-
-    }
-    );
-
-    after(function () {
+var signinBody = {
+    "username":"test",
+    "password":"password"
+}
 
-    }
-    );
+describe('Home page endpoint', function () {
+    it('should load the home page', function() {
+        return chai.request(app)
+        .get('/')
+        .then(function(res) {
+            expect(res).to.have.status(200);
+        })
+    })
+})
 
+describe('Registration endpoint', function () {
     // POST register fake account.
-    it('should register test user', function() {
+    it('should try and register user', function() {
         return chai.request(app)
         .post('/auth')
-        .send(body)
+        .send(registrationBody)
         .then(function(res) {
-            expect(res).to.have.status(200);
-            expect(res).to.be.json;
-            expect(res.body).to.be.an('object');
-            expect(res.body.results).to.be.an('array');
+            expect(res).to.have.status(200)
           });
     });
 }
 )
```

Comment:
This is the sign in endpoint test. Since the endpoint hasnt been implemented, I cannot run the test yet.